dessia_common/templates.py

```diff
@@ -322,14 +327,18 @@
 ''')
 
 
-Dataset_markdown_template = Template('''
-# Object $name of class $class_
+dataset_markdown_template = Template('''
```

Comment:
THANK YOU ! What a horrible mess of a variable name it was

Reply:
mdr